modules/ROOT/pages/to-search-with-graph-api.adoc

```diff
@@ -4,39 +4,61 @@ include::_attributes.adoc[]
 
-*Notes:*
+The Exchange Graph API can be used without an Anypoint Platform account to find assets made publicly available by MuleSoft. Accessing private content through the API requires an Anypoint Platform account.
 
-* The Exchange Graph API can be used without an Anypoint Platform account. In that case, only assets made publicly available by MuleSoft are returned.
-* To utilize this API to access private content, you must be provisioned with an Anypoint Platform account.
+== Authentication
 
-== Obtain an Access Token
+Use either an access token or a connected application for authentication.
 
-You only need an access token if you are searching for assets that are not created by MuleSoft.
+You only need authentication if you are searching for assets that are not created by MuleSoft.
 
-Access tokens expire sometime between 15 to 180 minutes (ask your organization administrator), so you may need to get a new token while doing searches.
+To use token authentication:
 
-. Log into Anypoint Platform.
-. Obtain an access token with a curl command such as the following. You can also use Postman or another application for sending HTTP commands.
-+
-[source,json,linenums]
+. Log in to Anypoint Platform.
+. Obtain an access token with a cURL command.
+
+You can also use Postman or another application for sending HTTP commands.
+
+Replace `ANYPOINT_USERNAME` with your Anypoint Platform user account name and replace `ANYPOINT_PASSWORD` with your password.
+
+[source,console]
 ----
-curl -X POST \
-  https://anypoint.mulesoft.com/accounts/login \
-  -H 'content-type: application/json' \
-  -d '{
-  "username": "YOUR_ANYPOINT_PLATFORM_USER_NAME",
-  "password": "YOUR_ANYPOINT_PLATFORM_PASSWORD"
-}'
+curl --location --request POST 'https://anypoint.mulesoft.com/accounts/login' \
+     --header 'Content-Type: application/json' \
+     --header 'Accept: application/json' \
+     --data-raw '{
+        "username":"ANYPOINT_USERNAME",
+        "password":"ANYPOINT_PASSWORD"
+     }' | jq -r ".access_token"
 ----
 
+Use this access token in your API calls.
+
+Your organization administrator sets the expiration time for access tokens to a value from 15 to 180 minutes. If you perform a search or other operation that requires more time, you must request an additional token.
+
 See also https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/api/v1/pages/Authentication/[Access Management API Authentication].
 
+To use connected application authentication:
+
+. xref:connected-app-authentication.adoc[Create a connected application].
+. Provide basic authentication and define the username as `\~~~Client\~~~` and the password as `clientId\~?~clientSecret`.
+.. Replace `clientId` with the client ID.
+.. Replace `clientSecret` with the client secret.
+. Define the connected application authentication as an authorization field in the query variables:
```

Comment:
After Step 1, the user should do either 2.Basic Auth or 3.Query Variable. Currently it reads like they would need to do both. It would worth adding if both are used, the Basic Auth takes precedence